modules/ServerAPI/src/server/routes/PreferredEntityCodeService.coffee

```diff
@@ -57,6 +61,38 @@ exports.getSpecificEntityType = (codeNameOrDisplayName, callback) ->
 	else
 		return entityType
 
+exports.getConfiguredTestedEntityTypesRoute = (req, resp) ->
+	if req.params.asCodes?
+		asCodes = true
+	else
+		asCodes = false
+	testedEntityTypes = exports.getConfiguredTestedEntityTypes asCodes
+	resp.json testedEntityTypes
+
+exports.getConfiguredTestedEntityTypes = (asCodes) ->
+	testedEntityTypes = _.where configuredEntityTypes.entityTypes, {isTestedEntity: true}
+	if asCodes
+		codes = for type in testedEntityTypes
+			code: type.code
+			name: type.displayName
+			ignored: false
+		return codes
+	else
+		return testedEntityTypes
+
+exports.getSpecificLoableEntityTypeRoute = (req, resp) ->
```

Comment:
What is this function name? Seems like it should be exports.getSpecificTestedEntityTypeRoute